src/xtopcom/xdata/src/xtransaction_v3.cpp

```diff
@@ -25,18 +26,19 @@ namespace top { namespace data {
 
 using namespace top::base;
 
-xtransaction_v3_t::xtransaction_v3_t() {
+xtransaction_v3_t::xtransaction_v3_t() : xstatistic::xstatistic_obj_face_t(xstatistic::enum_statistic_tx_v3) {
```

Comment:
should the inheritance be ENABLED by defining CACHE_SIZE_STATISTIC or CACHE_SIZE_STATISTIC_MORE_DETAIL?

Reply:
no, there is an empty base class for not defining CACHE_SIZE_STATISTIC or CACHE_SIZE_STATISTIC_MORE_DETAIL.